src/extension/wps/wps-core/src/test/java/org/geoserver/wps/gs/ImportProcessTest.java

```diff
@@ -31,6 +35,10 @@
     @After 
     public void removeNewLayers() {
         removeLayer(SystemTestData.CITE_PREFIX, "Buildings2");
+        removeLayer(SystemTestData.CITE_PREFIX, "Buildings4");
+        removeLayer(SystemTestData.CITE_PREFIX, "Buildings5");
+        removeStore(SystemTestData.CITE_PREFIX, SystemTestData.CITE_PREFIX + "data");
+        removeStore(SystemTestData.CITE_PREFIX, SystemTestData.CITE_PREFIX + "raster");
```

Comment:
I cannot remember how specific the CITE test data is (like is it only for CITE testing).

I guess in this case you are just cleaning up after layers you may of added during testing ...